crates/context/src/lib.rs

```diff
@@ -400,6 +401,7 @@ impl ContextManager {
 
         let Some(ContextIdentityValue {
             private_key: Some(requester_secret),
+            sender_key: None,
```

Comment:
we shouldn't care about `sender_key` here, and if we did it's almost a 100% certainty that when `private_key` is `Some`, `sender_key` will be too.

```suggestion
            ..
```